.github/workflows/cd.yml

```
@@ -23,29 +24,38 @@ jobs:
 
       - name: Deploy to Compute Engine
         run: |
-          # SSH 키 설정
-          echo "${{ secrets.GCP_SSH_PRIVATE_KEY }}" > ./private_key
-          chmod 600 ./private_key
+          # 1. SSH 키를 파일로 생성
+          echo "${{ secrets.GCP_SSH_PRIVATE_KEY }}" > ./gcp_ssh_key
+          chmod 600 ./gcp_ssh_key
           
```

YAMLlint (1.37.1)

[error] trailing spaces

(trailing-spaces)